configs/stdc/fcn_stdc-813_512x1024_40k_cityscapes.py

```diff
@@ -0,0 +1,4 @@
+_base_ = [
+    '../_base_/models/fcn_stdc-813.py', '../_base_/datasets/cityscapes.py',
```

Comment:
We would rename `813` in the future cause it means size of original repo .pth file is 813Mb.